### ISSUE TITLE
 Blynk.begin(auth) verses Blynk.begin(auth,ssid,pass) 

### ISSUE THREAD

**Issue by captainchaosrules** [tzapu/WiFiManager#214]
Trying to get code to work with Blynk
I get this error when using just Blynk.begin(auth)
aug12b_CONFIG_BLYNK_git:52: error: no matching function for call to 'BlynkWifi::begin(char [33])'
If I use this: 
Blynk.begin(auth,ssid,pass)
how would I pass variables ssid and pass?
# include <ESP8266WiFi.h>          //https://github.com/esp8266/Arduino

//needed for library
# include <DNSServer.h>
# include <ESP8266WebServer.h>
# include <WiFiManager.h>          //https://github.com/tzapu/WiFiManager
# include <BlynkSimpleEsp8266.h>

/////////////////////////
char auth[] = "13a164ce23a04f388328501154c4b60b";

// Your WiFi credentials.
// Set password to "" for open networks.

//char ssid[] = "test";
//char pass[] = "testest";

///////////////////////////

//for LED status
# include <Ticker.h>

Ticker ticker;

void tick()
{
  //toggle state
  int state = digitalRead(BUILTIN_LED);  // get the current state of GPIO1 pin
  digitalWrite(BUILTIN_LED, !state);     // set pin to the opposite state
}

//gets called when WiFiManager enters configuration mode
void configModeCallback (WiFiManager *myWiFiManager) {
  Serial.println("Entered config mode");
  Serial.println(WiFi.softAPIP());
  //if you used auto generated SSID, print it
  Serial.println(myWiFiManager->getConfigPortalSSID());
  //entered config mode, make led toggle faster
  ticker.attach(0.2, tick);
}

void setup() {
  // put your setup code here, to run once:

Serial.begin(115200);

  //Blynk.begin(auth, ssid, pass);
  Blynk.begin(auth);

  //set led pin as output
  pinMode(BUILTIN_LED, OUTPUT);
  // start ticker with 0.5 because we start in AP mode and try to connect
  ticker.attach(0.6, tick);

  //WiFiManager
  //Local intialization. Once its business is done, there is no need to keep it around
  WiFiManager wifiManager;
  //reset settings - for testing
  //wifiManager.resetSettings();

  //set callback that gets called when connecting to previous WiFi fails, and enters Access Point mode
  wifiManager.setAPCallback(configModeCallback);

  //fetches ssid and pass and tries to connect
  //if it does not connect it starts an access point with the specified name
  //here  "AutoConnectAP"
  //and goes into a blocking loop awaiting configuration
  if (!wifiManager.autoConnect()) {
    Serial.println("failed to connect and hit timeout");
    //reset and try again, or maybe put it to deep sleep
    ESP.reset();
    delay(1000);
  }

  //if you get here you have connected to the WiFi
  Serial.println("connected...yeey :)");
  ticker.detach();
  //keep LED on
  digitalWrite(BUILTIN_LED, HIGH);
}

void loop() {
  // put your main code here, to run repeatedly:

//Blynk.begin(auth, ssid, pass);

//Blynk.begin(auth);
Blynk.run();
}

**Comment by captainchaosrules** [comment]
This seemed to solve the issue...
Blynk.config(auth);

**Comment by mridzkyf** [comment]
> This seemed to solve the issue... Blynk.config(auth);

Hi this method doesn't work on my project? I wonder if there is another way to blynk.begin without ssid and pass
